fe/fe-core/src/main/java/com/starrocks/external/elasticsearch/EsNodeInfo.java

```diff
@@ -100,6 +100,24 @@ public EsNodeInfo(String id, Map<String, Object> map) throws StarRocksESExceptio
         }
     }
 
+    public EsNodeInfo(String id, String seed) {
```

Comment:
Why the second arg name is seed?

Reply:
I haven't verified this PR in the cluster, this parameter is wrong，I would fix it now

Reply:
The calling place has looped the String array, and the element inside is ip:port

Reply:
The calling place has looped the String array, and the element inside is ip:port，one EsNodeInfo one seed